arabic/salamoAlikom.md

```diff
@@ -15,3 +15,7 @@
 [وظيفة](Job/Job.md)
 
 [أريد أن أحصل على Nanodegree](https://www.udacity.com/nanodegree)
+
+ [online](earn/learn.md) عايز اعرف اسماء مواقع  كورسات  
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.